src/components/dashboard/chores/Chores.js

```diff
@@ -0,0 +1,316 @@
+import React, { Component } from 'react';
+import '../chores/chores.css';
+import M from "materialize-css";
+const baseUrl = process.env.REACT_APP_BASE_URL || 'http://localhost:8080';
+
+class Chores extends Component {
+    state = {
+        detailsBool: false,
+        choreAssigneeUsername: '',
+        choreCategoryId: '',
+        choreName: '',
+        choreDateComplete: '',
+        choreDifficulty: '',
+        choreFreqId: '',
+        choreNotes: '',
+        choreStatus: ''
+    }
+    user = JSON.parse(localStorage.getItem('user'));
+    usersCategories = [];
+    frequencies = [];
+    currentChore = {};
+
+    componentDidMount() {
+        this.getUsersCategories();
+        this.getFrequencies();
+        M.AutoInit();
+    }
+
+    handleChoreClick(event, modal, chore) {
+        event.preventDefault();
+        let elem = document.querySelector(modal);
+        this.getUserById(chore.assignee_id);
+        this.currentChore = chore; 
+
+        if(!chore.notes) {
```

Comment:
Could do something like this: 

```
if(!chore.notes) {
    chore.notes = '';
}

this.setState({ 
                choreCategoryId: chore.category_id,
                choreName: chore.chore_name,
                choreDateComplete: chore.date_complete,
                choreDifficulty: chore.difficulty,
                choreFreqId: chore.frequency_id,
                choreNotes: chore.notes,
                choreStatus: chore.status,
                detailsBool: !this.state.detailsBool
});
```

Reply:
Good job!  This should be `chore.notes = ''` instead because here you will set the state for choreNotes and then after the if you re-set the state of choreNotes on line 45

Reply:
@natalievasquez11 ^